tmail-backend/mailets/src/test/scala/com/linagora/tmail/mailets/ContactsCollectionTest.scala

```diff
@@ -101,6 +101,26 @@ class ContactsCollectionTest {
       .containsExactlyInAnyOrder(ContactFields(new MailAddress(RECIPIENT)))
   }
 
+  @Test
+  def serviceShouldDispatchEventWhenHasInvalidRecipient(): Unit = {
+    mailet.init(MAILET_CONFIG)
+
+    val mail: FakeMail = FakeMail.builder()
+      .name("mail1")
+      .mimeMessage(MimeMessageBuilder.mimeMessageBuilder()
+        .setSender(SENDER)
+        .addToRecipient("recipient", RECIPIENT3)
```

Comment:
```suggestion
        .addToRecipient("invalidRecipient", RECIPIENT3)
```

would be more understandable IMO.